xbmc/windowing/windows/WinKeyMap.h

```diff
@@ -172,7 +173,7 @@ namespace KODI
           VK_keymap[VK_LAUNCH_APP1] = XBMCK_LAUNCH_APP1;
           VK_keymap[VK_LAUNCH_APP2] = XBMCK_LAUNCH_APP2;
         }
-      }
+    }
```

Comment:
This looks thrown off with the rest of the whitespace. May be best to clang-format everything past the ifdef VK_PRINT

Reply:
I decided to add a pre-commit that runs clang-format on the entire file. I think that makes the diff look a lot better.